### DISCUSSION TITLE
Can't see my achievement badge Starstruck

### DISCUSSION THREAD

**Discussion by vm32** [community/community#81809] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Bug

### Body

I have a repo with 56 stars and it does not show in the Starstruck !!

**Comment by T9ipa** [comment]
Oh

**Comment by brf153** [comment]
@vm32 you already have one starstruck in your profile. You will get another starstruck(bronze one) when one of your repos get 128 stars. for more info visit https://github.com/Schweinepriester/github-profile-achievements

**Comment by faridnec** [comment, marked as answer]
I got the same issue with my badge too but then I tried to click my achievements. It showed up the list of badges I got and there is there. So I hide it by clicking hide and then unhide it again to show. It worked and my bagde finally shows up under my profile.

Sometimes it does like that because maybe this feature is still in beta (?) I hope this might help your issue

**Reply by KnightChaser** [reply]
Thanks for your answer. I also faced a similar trouble with that badge and resolved the issue by referring to your answer. :+1:
